content/docs/reactivesearch/searchbox/Quickstart.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'QuickStart'
+meta_title: 'QuickStart to SearchBox'
+meta_description: 'SearchBox is a lightweight search library with some common utilities.'
+keywords:
+    - quickstart
+    - react-searchbox
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'searchbox'
+---
+
+[searchbox](https://github.com/appbaseio/searchbox) - A lightweight search library with some common utilities.
```

Comment:
@jyash97 See the description for React Searchbox. This is for ElasticSearch, that's not clear in your description.